core/templates/cards.html

```diff
@@ -75,27 +75,28 @@ <h4 class="modal-title">Col-League</h4>
                     to upload an image and become part of the game!
                     </p>
                 </div>
-                <div class="modal-footer">
-                    <button id="nextRound" onclick="startGame()" class="btn btn-primary pull-left" data-dismiss="modal">I don't care   <i class="fa fa-meh-o" ></i></button>
-                    <a class="btn btn-default" href="{% url 'auth:logout' %}?next=/">Logout</a>
-                </div>
             {%  elif '/no_photo.png' not in request.user.yammer_url  %}
-                 <div class="modal=body">
+                <div class="modal=body">
```

Comment:
is the `=` a typo, or intentional?  I'm a CSS noob so I'm curious if there's a meaning behind it if intentional.